types/meteor/tsconfig.json

```diff
@@ -27,4 +27,4 @@
         "test/react-meteor-data-tests.tsx",
         "test/meteor-mongo-tests.ts"
     ]
-}
+}
```

Comment:
I'd revert this change for two reasons:
1. We'll need to go through a much longer review to merge this PR.
2. All other files have an additional new line at the end.